dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -605,28 +606,28 @@ public String getReasonPhrase() {
      * Return a matching source file
      *
      * @param sourceFiles      files to look through
-     * @param searchPathParam       file to look for
+     * @param searchPathParam       file to look for, could be relative or absolute
      * @param workingDirectory working directory if relevant
      * @return
      */
     public Optional<SourceFile> lookForFilePath(Set<SourceFile> sourceFiles, String searchPathParam, String workingDirectory) {
-        // ignore leading slashes
-        String searchPath = cleanRelativePath(searchPathParam);
-
-        for (SourceFile sourceFile : sourceFiles) {
-            String calculatedPath = sourceFile.getAbsolutePath();
-            // annoyingly, test json and Dockerfiles include a fullpath whereas descriptors are just relative to the main descriptor,
-            // so we need to standardize relative to the main descriptor
-            if (SourceFile.TEST_FILE_TYPES.contains(sourceFile.getType())) {
-                calculatedPath = StringUtils.removeStart(cleanRelativePath(sourceFile.getPath()), cleanRelativePath(workingDirectory));
-            }
-            calculatedPath = cleanRelativePath(calculatedPath);
-            if (searchPath.equalsIgnoreCase(calculatedPath) || searchPath
-                .equalsIgnoreCase(StringUtils.removeStart(calculatedPath, workingDirectory + "/"))) {
-                return Optional.of(sourceFile);
-            }
+        String targetPath;
+        if (searchPathParam.startsWith("/")) {
+            // treat searchPath as an absolute path
+            targetPath = cleanRelativePath(searchPathParam).toLowerCase();
```

Comment:
I had some tests fail when I turned that on. I'll take a closer look, but I was wary of adding an additional backwards incompatible change in case cromwell, cwl are not case sensitive.
Or if they run on Windows for that matter

Reply:
Yeah, probably worth pursuing; it's a real edge case, and there are various factors at play; not just the OS, but how you configure git locally. Just mentioned it because I've been bitten by this a few times, but only over a very long period of time.

Reply:
Took a brief look at this, I think the tests that are breaking are because the casing of the descriptor path specified in the default workflow parameters did not match the casing here. 

In other words, if we make things case-sensitive here then we need to treat things as case sensitive in the UI when entering paths and in .dockstore.yml. I'm thinking that this can be broken out into a separate ticket if we do make this change in behaviour